packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/EnterAmount.tsx

```diff
@@ -354,26 +354,28 @@ const Account = ({
   )
 }
 
-const NextButton = ({ invalid, orderType, paymentAccount, pristine, submitting }) => {
-  return (
-    <Button
-      data-e2e={orderType === BrokerageOrderType.DEPOSIT ? 'submitDepositAmount' : 'withdrawNext'}
-      height='48px'
-      size='16px'
-      nature='primary'
-      type='submit'
-      fullwidth
-      disabled={invalid || pristine || submitting || !paymentAccount}
-      onClick={() => {}}
-    >
-      {submitting ? (
-        <HeartbeatLoader height='16px' width='16px' color='white' />
-      ) : (
-        <FormattedMessage id='buttons.next' defaultMessage='Next' />
-      )}
-    </Button>
-  )
-}
+const PreviewButton = ({ invalid, orderType, paymentAccount, pristine, submitting }) => (
+  <Button
+    data-e2e={orderType === BrokerageOrderType.DEPOSIT ? 'submitDepositAmount' : 'withdrawNext'}
+    height='48px'
+    size='16px'
+    nature='primary'
+    type='submit'
+    fullwidth
+    disabled={invalid || pristine || submitting || !paymentAccount}
+    onClick={() => {}}
```

Comment:
I see that this wasn't on the old button either, but is this an issue? Does the form still submit even if there is no onclick handler?

Reply:
its on the old button.

Reply:
Ok yeah I think since it's `type='submit'` the form just submits when the user clicks it so this works.